readme.md

```diff
@@ -52,9 +52,17 @@ Number of digits to appear after the seconds decimal point.
 ##### compact
 
 Type: `boolean`
```

Comment:
And do you see a difference between how the `secDecimalDigits` content looks compared to `verbose` and `compact`?

> What do you mean by needs to spaces at the end ?

Exactly what I said. 2 spaces is how you make a linebreak in markdown.